magento/pwa-studio
# [PWA-2554] Long wishlist name breaks layout

## Description

Fixed overflow of text for custom wishlist name

## Related Issue

Closes PWA-2554

## Acceptance

### Verification Stakeholders

Project Maintainers

### Specification




## Verification Steps






Wishlist should account for custom names that are too long for the viewable UI.

#### Test scenario(s) for direct fix/feature





1. Log into Venia
2. Navigate to "Favorite List"
3. Add a custom  favorite list name that is super long
4. Notice that the name should now truncate when it reaches the end of the line, but will have a hover over with the full name

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations

N/A




1. Verify that new and existing Jest tests pass
1. Verify that Cypress test passes

#### Is Browser/Device testing needed?





Yes

#### Any ad-hoc/edge case scenarios that need to be considered?

No




## Screenshots / Screen Captures (if appropriate)



## Breaking Changes (if any)





No breaking changes

## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
